setup.py

```diff
@@ -44,4 +44,5 @@
 setup(
     ext_modules=cython_extensions + pybind_extensions,
     cmdclass={"build_ext": build_ext},
+    package_data={'tools': ['RAiDER/models/*.zip']}
```

Comment:
@jhkennedy - can you confirm this is done correctly?

I am using https://docs.python.org/3.11/distutils/setupscript.html#installing-package-data

Want to include a zipped file as data.